packages/store/src/decorators/select/symbols.ts

```diff
@@ -0,0 +1,45 @@
+import { Observable } from 'rxjs';
+
+import { CONFIG_MESSAGES, VALIDATION_CODE } from '../../configs/messages.config';
+import { propGetter } from '../../internal/internals';
+import { SelectFactory } from './select-factory';
+import { META_KEY } from '../../symbols';
+
+const DOLLAR_CHAR_CODE = 36;
+
+/**
+ * The SelectUtils namespace is used for logical grouping of functionalities.
+ */
+export namespace SelectUtils {
+  export function selectBySelector<T = any>(selector: any): Observable<T> {
+    if (!SelectFactory.store) {
+      throw new Error(CONFIG_MESSAGES[VALIDATION_CODE.SELECT_FACTORY_NOT_CONNECTED]());
+    }
+
+    return SelectFactory.store.select(selector);
+  }
+
+  export function unwrapSelector(name: string, rawSelector?: any, paths: string[] = []): any {
```

Comment:
Maybe this could have a better name?
possibly `createSelectorFn`

Reply:
done